tetris.py

```diff
@@ -312,20 +313,32 @@ def __init__(self, main_board, topbar_time):
         self.main_board = main_board
         self.topbar_time = topbar_time
         self.start_time = time.time()
+        self.paused = False
         self.timer()
 
+    def pause_game(self, event):
+        if self.paused is True:
+            self.paused = False
+            self.move_block_down()
+        else:
+            if not self.paused:
+                self.paused = True
```

Comment:
No need to check anything, because the `else` never runs with `self.paused` set to `True`:
```suggestion
            self.paused = True
```